### ISSUE TITLE
Dogecoin Core Design Suggestion

### ISSUE THREAD

**Issue by Maniixer** [dogecoin/dogecoin#2948]
Made a re-design of the Dogecoin Core wallet with some minor changes, to make it maybe a little more appealing :D
![Dogecoin Core design](https://user-images.githubusercontent.com/92598085/167434264-543559ad-4989-40b6-98d5-93cbeb412342.png)

**Comment by maryamMaryam09367011915** [comment]
maryam.jahani19@gmail.com

**Comment by maryamMaryam09367011915** [comment]
09367011915

**Comment by Maniixer** [comment]
any feedback is welcome, mostly wondering what is possible to change as style in the wallet as i have no clue how it is made.

**Comment by kevstar0** [comment]
I support this idea

**Comment by jadehamel** [comment]
Hello! I have a question about your UI/UX proposition which I understand having as goal to be more appealing. I agreed with your pov and representation, but I would split the marketing argue about it. As I think roght now, If you want to upgrade the doge wallet ui/ux for markething reasons is opposite to it's first reason to exists.... Otherwise I think your point is great but why not just fork the wallet et make a new one with a different philo!?

**Comment by Maniixer** [comment]
Had no marketing reason in my mind for wanting to give some ideas for a kinda newer look on the wallet as it looks like its a program from the 90'😆 and your right they should make a design as how dogecoin is represented and they probably are doing that because that would be such wow haha. But i'm just someone from the community who likes dogecoin, and would love to help where i can, as i am not involved in the dev/design team.

**Comment by cezardoob** [comment]
Dark mode would be much appreciated

**Comment by mocherz** [comment]
much nice!

**Comment by Maniixer** [comment]
![image](https://user-images.githubusercontent.com/92598085/177144580-25f02081-5ffb-4825-9460-a831f516ba80.png)

**Comment by Maniixer** [comment]
Changed input field Order  "Amount" with "Message" for the rest no layout change, just a bit of styling.
![image](https://user-images.githubusercontent.com/92598085/177313332-1769f95e-47fd-4f08-8935-bf0f7dfb42c4.png)

**Comment by Maniixer** [comment]
removed the bold font on recent transactions, and gave the "Tip of the day" some extra attention.
![image](https://user-images.githubusercontent.com/92598085/177314559-ec35137e-ef33-4ed0-affb-26f6bb09dd4e.png)

**Comment by daank-c** [comment]
this is awesome. Is there anyway to toggle the decimals ^ .^

**Comment by daank-c** [comment]
Absolutely. Even a little [show more/show less] toggle next to the header of the Recent Transactions.
![177314559-ec35137e-ef33-4ed0-affb-26f6bb09dd4e](https://user-images.githubusercontent.com/102710924/177318744-b0c1b7a8-7f74-42ed-96c6-67b46bc0ec4d.png)
.

**Comment by Maniixer** [comment]
I think it would have a better place next to the "Balances" section, instead of the "Recent transactions" as that would tell the user to show more transactions. Right now if you klick on any of the recent transactions it will bring you to the "Transactions TAB" where all the made transactions are shown.

Also showing less or more decimals should only be shown on the "Home" page and not any other tab, as they are important to see every detail.

**Comment by Maniixer** [comment]
Darkmode! :p 
![image](https://user-images.githubusercontent.com/92598085/177330137-bc23e333-37ad-488a-8229-2b40a7fdc7cf.png)

**Comment by Maniixer** [comment]
Don't forget about dark mode! 😋

![image](https://user-images.githubusercontent.com/92598085/177338252-747260a4-f701-4ae6-b9f0-ab545f6f33ba.png)
![image](https://user-images.githubusercontent.com/92598085/177339357-f86eb2f0-fb1a-4464-8516-99c1c7d7563f.png)
![image](https://user-images.githubusercontent.com/92598085/177339394-58a86480-6751-4bf5-a0cc-84b7d814cebd.png)

**Comment by Maniixer** [comment]
Good note! 
It can be fixed by making the text less white and slightly more grey.

**Comment by Maniixer** [comment]
Placed a Light - Dark mode switch, but having the option in the option menu might be a better place as it it does not need extra GUI work.
https://user-images.githubusercontent.com/92598085/178143315-fa303239-b30b-433f-ae68-b29ffcaab13c.mp4

**Comment by patricklodder** [comment]
> would you think its possible to add this amount of styling to the wallet, or would it be limited to how the wallet has been coded?

I think all of this is possible with the way the current code is structured, either after we update 1.14 to a more recent QT, or on 1.21.

**Comment by alamshafil** [comment]
I can help setting up dark/light mode and creating the QT style-sheets. I already have code I can use for the dark/light mode; however, it only applies after a client restart.

**Comment by alamshafil** [comment]
Didn't know about! That's good to know.

**Comment by Maniixer** [comment]
Added some feature ideas, not sure if possible to add code to show the value of the amount of doge :p (probably not needed as u can see more details somewhere else)
![image](https://user-images.githubusercontent.com/92598085/180601841-1ab38ccf-2631-4a07-b457-945658c371c5.png)

**Comment by patricklodder** [comment]
> not sure if possible to add code to show the value of the amount of doge 

I'm not convinced that it's desirable but I'm reluctant to start the discussion here because it will be opinionated and off-topic. If you think it's a good idea to add this feature, please [open a separate feature request](https://github.com/dogecoin/dogecoin/issues/new?template=feature_request.md) or [start a discussion](https://github.com/dogecoin/dogecoin/discussions/new)

**Comment by dommexx20** [comment]
> Made a re-design of the Dogecoin Core wallet with some minor changes, to make it maybe a little more appealing :D ![Dogecoin Core design](https://user-images.githubusercontent.com/92598085/167434264-543559ad-4989-40b6-98d5-93cbeb412342.png)
